modules/nw-router-configuring-dual-stack.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * networking/routes/route-configuration.adoc
+
+[id="nw-router-configuring-dual-stack_{context}"]
+= Configuring the {product-title} Ingress Controller for dual-stack support
+
+When you install a {product-title} cluster on a IPv4/IPv6 dual-stack enabled bare metal infrastructure, your cluster to is externally reachable by {product-title} routes.
```

Comment:
I believe it is configurable at install time and [afterward](https://docs.openshift.com/container-platform/4.8/networking/ovn_kubernetes_network_provider/converting-to-dual-stack.html#converting-to-dual-stack).

If you are open to suggestion, I think we should use "IPv4 and IPv6" rather than the slash--on that other page too, that page is making me edgy.

"dual-stack enabled bare metal infrastructure" seems like a word tangle.  Maybe "If your {product-title} cluster is configured for IPv4 and IPv6 dual-stack networking, your cluster is..."  (the "to is" in the second part leaves me unsure what you want to say about the routes).